config/routes.rb

```diff
@@ -816,23 +816,22 @@
 
     scope '/performance' do
       get '/' => 'performance#index', as: :performance
-      get '/dashboard' => 'performance_dashboard#dashboard', as: :performance_dashboard
 
-      get '/course-stats', to: 'performance#course_stats', as: :course_stats
+      get '/dashboard' => 'performance#performance_dashboard', as: :performance_dashboard
```

Comment:
A dashboard among dashboards — could this be `service-performance` and `service_performance_dashboard` maybe?

Reply:
How does `/support/performance/service` grab you?

Reply:
🙆‍♂️ !